cmp/grid/renderers/MultiFieldRenderer.ts

```diff
@@ -107,13 +116,19 @@ function renderSubField({colId, label}, context) {
 
     if (label && !isString(label)) label = headerName;
 
-    return div({
-        className: 'xh-multifield-renderer-field',
-        items: [label ? `${label}: ` : null, renderValue(value, renderer, column, context)]
-    });
+    const renderedVal = renderValue(value, renderer, column, context),
+        renderedValIsEmpty = renderedVal === '' || isNil(renderedVal);
```

Comment:
It seems like allowing '' to render might be useful to allow applications that did want to show the label for an "empty" value a way to do so